doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst

```diff
@@ -245,3 +254,118 @@ The following samples allow you to test if the buttons (or switches) and LEDs on
 
 Build and flash the samples to make sure |NCS| is running correctly on your development kit.
 For the button and LED definitions, see the :file:`boards/arm/nrf7002dk_nrf5340/nrf5340_cpuapp_common.dts` file.
+
+Shields and Expansion Boards
+****************************
+
+Shields and expansion boards are add-on hardware that can be attached to a development kit or prototyping platform to extend their features and functionalities.
+
+nRF7002 EK
+==========
+
+The nRF7002 :term:`Evaluation Kit (EK)` is a versatile evaluation kit in the form of an Arduino shield.
+The kit can be used to provide Wi-Fi connectivity and Wi-Fi-based locationing to compatible development or evaluation boards through the nRF7002 Wi-Fi 6 companion :term:`Integrated Circuit (IC)`.
+
+Overview
+--------
+
+The nRF7002 EK (PCA63556) is designed to provide Wi-Fi connectivity and Wi-Fi (SSID) scanning capabilities via the nRF7002 companion IC to a compatible host development board.
+
+The nRF7002 EK features an Arduino shield form factor and interface connector that allows it to be used with Arduino compatible boards, such as the nRF52840 DK, nRF5340 DK, or nRF9160 DK.
```

Comment:
Do not we need active links to these DK product pages (or corresponding NCS doc pages)??

Reply:
Added links to the DK product pages.